### PR TITLE
Standardize 'Show' labels to manifest fields

### LINKED ISSUE #1690: Update "show" output
### Description of the new feature / enhancement

The show command should display the values in the manifest as the key / value pairs rather than the current names.

For example "Download URL" is not one of the manifest keys.

### Proposed technical implementation details

_No response_

### PR DESCRIPTION
**Pull request by Trenly**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Are you working against an Issue?
    - Resolves #1690

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/2311)

### CODE REVIEW

**Comment by denelon** [comment]
@Trenly

We had a discussion internally a while back related to the labels. Part of the reason we were exposing the exact "YAML" key values is because we weren't localizing the titles. I'm not sure how well this plays from a localization perspective.

Thoughts?

**Comment by Trenly** [comment]
> @Trenly
> 
> We had a discussion internally a while back related to the labels. Part of the reason we were exposing the exact "YAML" key values is because we weren't localizing the titles. I'm not sure how well this plays from a localization perspective.
> 
> Thoughts?

Hmmm. I'm not sure either. I think from a contributor perspective, I would rather see exact YAML keys (spaces added) as it provides simpler correlation between fields. From a user perspective, I would want the fields to be localized so I could understand what each of them were. 

However, I would say that there is a fairly clear link between the Yaml keys and the meaning of the key. From an english-First perspective, I don't see any reason not to update the keys. From a localized perspective, I can see how it could make the localization more difficult to understand.

**Comment by yao-msft** [comment]
I think last time we discussed that we would de-localize the manifest keys for show, for scenarios Demitrius mentioned. Are we ok not doing that now?

File: src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw [hunk 1]
```diff
@@ -1055,7 +1055,7 @@ Do you agree to the terms?</value>
     <value>Moniker:</value>
   </data>
   <data name="ShowLabelPackageUrl" xml:space="preserve">
-    <value>Homepage:</value>
+    <value>Package Url:</value>
```
Comment (JohnMcPMS) [inline src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw:1058]: I think that this one could be confusing, as some people may think that "package" means "installer".

Comment (Trenly) [reply src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw]: Apologies for the delay; Reverted

**Comment by JohnMcPMS** [comment]
Not that a build is needed... just following protocol.

**Review by yao-msft — APPROVED** [review]

**Pull request merged**